internal/provider/resource_cloudflare_total_tls.go

```diff
@@ -0,0 +1,62 @@
+package provider
+
+import (
+	"context"
+	"fmt"
+	"github.com/cloudflare/cloudflare-go"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceCloudflareTotalTLS() *schema.Resource {
+	return &schema.Resource{
+		Schema:        resourceCloudflareTotalTLSSchema(),
+		CreateContext: resourceCloudflareTotalSSLUpdate,
+		ReadContext:   resourceCloudflareTotalSSLRead,
+		UpdateContext: resourceCloudflareTotalSSLUpdate,
+		DeleteContext: resourceCloudflareTotalSSLDelete,
```

Comment:
we'll want to add a description here.

Reply:
@jacobbednarz it seems the description was added :)